tests/lib/c_lib/thrd/testcase.yaml

```diff
@@ -0,0 +1,29 @@
+common:
+  tags:
+    - clib c11 threads
+  filter: not CONFIG_NATIVE_APPLICATION
+  integration_platforms:
+    - mps2_an385
+tests:
+  libraries.libc.c11_threads.minimal:
+    tags: minimal_libc
```

Comment:
Nowadays this will default to picolibc, so this would need to specifically select CONFIG_MINIMAL_LIBC

Reply:
Heh.. shows how long this one has been in development 🤣